cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -173,6 +173,7 @@ public static Kafka createKafka(String namespace, String name, int replicas,
                         .withLivenessProbe(probe)
                         .withReadinessProbe(probe)
                         .withStorage(new EphemeralStorage())
+                        .withVersion("3.0.0")
```

Comment:
Consider using `KafkaVersionTestUtils` to avoid hard coding a particular version.